spec/locales-currencies-tz.html

```diff
@@ -213,6 +222,29 @@ <h1>DefaultTimeZone ( )</h1>
         The DefaultTimeZone abstract operation returns a String value representing the valid (<emu-xref href="#sec-isvalidtimezonename"></emu-xref>) and canonicalized (<emu-xref href="#sec-canonicalizetimezonename"></emu-xref>) time zone name for the host environment's current time zone.
       </p>
     </emu-clause>
+
+    <emu-clause id="sec-availablecanonicaltimezones" type="implementation-defined abstract operation">
+      <h1>
+        AvailableCanonicalTimeZones (
+        ): a List of Strings
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>The returned List is a sorted List of supported Zone and Link names in the IANA Time Zone Database.</dd>
+      </dl>
+
+      <emu-alg>
+        1. Let _names_ be a List of all Zone and Link names in the IANA Time Zone Database that are supported by the implementation.
+        1. Let _result_ be a new empty List.
+        1. For each element _name_ of _names_, do
+          1. Assert: ! IsValidTimeZoneName( _name_ ) is *true*.
+          1. Let _canonical_ be ! CanonicalizeTimeZoneName( _name_ ).
+          1. If _result_ does not contain _canonical_, then
+            1. Append _canonical_ to the end of _result_.
+        1. Sort _result_ in order as if an Array of the same values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
```

Comment:
There's a lint issue here: https://github.com/tc39/ecma402/actions/runs/4137409543/jobs/7152541910?pr=716
> ```
> error: could not find a preceding declaration for "comparefn" (use-before-def) at spec/locales-currencies-tz.html:249:135:
>   247 |           1. If _result_ does not contain _canonical_, then
>   248 |             1. Append _canonical_ to the end of _result_.
> > 249 |         1. Sort _result_ in order as if an Array of the same values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
>       |                                                                                                                                       ^
>   250 |         1. Return _result_.
>   251 |       </emu-alg>
>   252 |     </emu-clause>
> ```

```suggestion
        1. [declared="comparefn"] Sort _result_ in order as if an Array of the same values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
```